core/services/head_tracker.go

```diff
@@ -496,19 +496,6 @@ func (ht *HeadTracker) isKovan() bool {
 	return ht.store.Config.ChainID().Cmp(kovanChainID) == 0
 }
 
-// totalNewHeadTimeBudget is the timeout on the shared context for all
-// requests triggered by a new head
-//
-// These values are chosen to be roughly 2 * block time (to give some leeway
-// for temporary overload). They are by no means set in stone and may require
-// adjustment based on real world feedback.
-func (ht *HeadTracker) totalNewHeadTimeBudget() time.Duration {
-	if ht.isKovan() {
-		return 8 * time.Second
-	}
-	return 26 * time.Second
-}
-
 // If total callback execution time exceeds this threshold we consider this to
 // be a problem and will log a warning.
 // Here we set it to the average time between blocks.
```

Comment:
What about totalExecutionTime below? We probably want to remove `isKovan` entirely.

Reply:
replaced totalExecutionTime call w/ direct config call for headTimeBudget

removed isKovan~